src/components/PatientList.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { parseString } from "xml2js";
+import { Tile } from "carbon-components-react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import styles from "./PatientList.module.scss";
+
+const PatientList = () => {
+  const [patients, setPatients] = useState([]);
+  const [activePatientCount, setActivePatientCount] = useState(0);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    try {
+      const response = await axios.get(
+        "/openmrs/ws/rest/v1/bahmnicore/sql?location_uuid=833d0c66-e29a-4d31-ac13-ca9050d1bfa9&q=emrapi.sqlSearch.activePatients",
```

Comment:
Hardcoded location UUID for now?

Reply:
The Location UUID is a system generated ID for a location, and it is not going to be the same for every bahmni instance / location. Are you going to play a different story later, to read the actual UUID of a location via API?

Reply:
Ok, I will hardcode it with the (location_uuid). So that anyone using it can put according to their location.

Reply:
The code needs to detect the location without any hardcoded location UUID. Can you do that?

Reply:
I can try that